code/modules/projectiles/guns/energy.dm

```diff
@@ -765,7 +763,7 @@
 		/obj/item/attachable/bayonetknife,
 		/obj/item/attachable/bayonetknife/som,
 		/obj/item/attachable/magnetic_harness,
-		/obj/item/attachable/scope/unremovable/laser_sniper_scope,
+		/obj/item/attachable/scope/mini,
```

Comment:
I am almost certain this hasn't been tested. laser sniper scope is hard built into the sprite.